src/utils/makeTempDir.ts

```diff
@@ -0,0 +1,10 @@
+import * as fse from "fs-extra";
+
+/**
+ *
+ * @description creates a directory
+ * @param pathToDir the path to the directory
+ */
+export function makeTempDir(pathToDir: string) {
+  return fse.ensureDir(pathToDir);
+}
```

Comment:
Not sure why we create proxy functions that execute one line of code? (makeTemplateDir, copyFile, checkFileExists, removeTempDir).

Reply:
I thought it would make it more readable rather than having `fse.someMethod` but I don't have a strong opinion. I'll refactor and remove them

Reply:
Done!